docs/bnb-smart-chain/introduction.md

```diff
@@ -64,13 +64,6 @@ BC and BSC share the same token universe for BNB and BEP2 tokens. This defines:
 2. The total circulation of the same token should be managed across the two networks, i.e. the total effective supply of a token should be the sum of the token’s total effective supply on both BSC and BC.
 3. The tokens can be initially created on BSC in a similar format as ERC20 token standard, or on BC as a BEP2, then created on the other. There are native ways on both networks to link the two and secure the total supply of the token.
 
-## Cross-Chain Transfer and Communication
```

Comment:
even the final sunset is not enabled
we can still not to emphasize the `Corss-Chain`